### DISCUSSION TITLE
How to create Nodes for inserting?

### DISCUSSION THREAD

**Discussion by JuroOravec** [rushter/selectolax#125] [General]
Hi, I want to use `Node. insert_after()`. Can I create an instance of `Node`, or do I need to use `HTMLParser(...).body.child` to create the node?

**Comment by rushter** [comment]
You can either pass Node instance or plain HTML as a string.

**Reply by JuroOravec** [reply]
Sorry, I wasn't clear, but how do I get the Node instance in the first place? Is there any other way than by `HTMLParser().body.child`? E.g. can I instantiate Node as `Node(*args, **kwargs)`?

**Reply by rushter** [reply]
It's not possible right now
